CHANGELOG.md

```diff
@@ -9,5 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `getEnumValueFromName`, `getEnumNames`, `getEnumNameValues` utils
+
+### Added
```

Comment:
@manni497 

There is already an unreleased `Added` section.

Reply:
Sorry I was working on a yesterday branch and I thought just to resolve the comment.. Now should be better :D